### DISCUSSION TITLE
Lighthouse report over mail

### DISCUSSION THREAD

**Discussion by pas-mayank-dewan** [GoogleChrome/lighthouse-ci#708] [Q&A]
Hi,
I am looking for a solution of how to send the final generated report over email. Is there a workaround to send the final .json or .HTML report over mail?

**Comment by AimalKhanOfficial** [comment]
What kind of a setup do you have? Is your LHCI setup built-in to your CI pipeline? Or is it local? FYI, i'm not on the official team for LHCI.
